bash/mz390

```diff
@@ -25,6 +25,7 @@ fi
 java -classpath ${jardir}/z390.jar -Xrs -Xms150000K -Xmx150000K $J2SEOPTIONS mz390 $1 $2 $3 $4 $5 $6 $7 $8 $9
 
 rc=$?
+# rc = 1 for Java issues; all other non-zero values for assembly issues
```

Comment:
message based on rc?

Reply:
No. Just documentation. Same as for ez390.java and lz390.java.